docs/reference/contributing/connectivity/LoRaPortingGuide.md

```diff
@@ -1,71 +1,69 @@
+Arm Mbed OS provides an API that serves as a template for any LoRa RF driver. [LoRaRadio](https://os-doc-builder.test.mbed.com/docs/development/mbed-os-api-doxy/class_lo_ra_radio.html) is a pure virtual class that standardizes the APIs across all LoRa radios. Mbed Enabled LoRa radio driver implementations present themselves as LoRaRadio. 
 
 <span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/lora_radio_inherit.png)<span>Figure 1. Existing Mbed LoRa RF drivers inherit from LoRaRadio class.</span></span>
 
-For a reference implementation, please check the existing [LoRa RF drivers](https://github.com/ARMmbed/mbed-semtech-lora-rf-drivers). Construction of a LoRaRadio object is a matter of taste. You may notice that the existing reference drivers allow construction of the LoRaRadio object with full pin definitions, that is to make sure that the driver is usable across platforms with any pin combination. You are free to use any form of construction as long as you provide a LoRaRadio object down to the Arm Mbed LoRaWAN stack. Utilization of an instance of `LoRaRadio` class for a 3rd party LoRaWAN stack is beyond the scope of this documentation but it should be trivial. 
+For a reference implementation, please check the existing [LoRa RF drivers](https://github.com/ARMmbed/mbed-semtech-lora-rf-drivers). Construction of a LoRaRadio object is a matter of taste. The existing reference drivers allow construction of the LoRaRadio object with full pin definitions. That is to make sure that the driver is usable across platforms with any pin combination. You are free to use any form of construction as long as you provide a LoRaRadio object down to the Arm Mbed LoRaWAN stack.
 
-For API use cases, details, explanation and meaning please check the `LoRaRadio` class reference given below. Data structures provided in [LoRaRadio.h](https://github.com/ARMmbed/mbed-os/blob/master/features/lorawan/LoRaRadio.h) are carefully planned and designed. They carry most of the stuff you may ever need to write your LoRa RF driver with.
+For API use cases, details, explanation and meaning, please see the `LoRaRadio` class reference at the end of this section. We carefully planned and designed the data structures that LoRaRadio provides. These data structure carry most of what you need to write your LoRa RF driver with.
 
-Although implementation details are left for the developers to ponder on, there are a few recommendations:
+Although implementation details are left to the developers, there are a few recommendations:
 
-1. The driver should support both RTOS and non-RTOS, i.e., the driver should be able to defer interrupts in case of RTOS using any of the Mbed-OS synchronization methods.
+1. The driver should support both RTOS and non-RTOS. In other words, the driver should be able to defer interrupts if the RTOS uses any of the Mbed OS synchronization methods.
```

Comment:
The meanings of the sentence have changed here. I would suggest to remove everything after non-RTOS.
This line could be removed 
```
 In other words, the driver should be able to defer interrupts if the RTOS uses any of the Mbed OS synchronization methods.
```

Developer will understand that he should support both RTOS and non-RTOS.